### PR TITLE
Support untilAttach param in history requests for realtime channels

### PR DESCRIPTION
**Pull request by SimonWoolf**

### CODE REVIEW

**Comment by SimonWoolf** [comment]
@mattheworiordan updated regen commit to include minified

File: browser/static/ably.js [hunk 1]
```diff
@@ -7693,6 +7703,42 @@ var RealtimeChannel = (function() {
 		this.pendingEvents = [];
 	};
 
+	RealtimeChannel.prototype.history = function(params, callback) {
```
Comment (paddybyers) [inline browser/static/ably.js]: It's impossible to see what changes are here because your editor is changing the whitespace in the file.

Comment (mattheworiordan) [inline browser/static/ably.js]: You use tabs no and this is a tab?

Comment (SimonWoolf) [inline browser/static/ably.js]: This is new - or rather, copied with changes from channel.js to realtimechannel.js. RealtimeChannel had no separate `history` method before. If you want to see the diff between `Channel#history` and `RealtimeChannel#history`, I've put it in a gist: https://gist.github.com/SimonWoolf/e4adee04837da359dc89

**Comment by SimonWoolf** [comment]
@paddybyers added https://github.com/ably/ably-js/commit/84e636febdb06a1fec07c754e07bbfc47e66b9c5 per your suggestion

**Comment by paddybyers** [comment]
Thanks, but please remove the commit to regenerate the static libs and I'll merge.

**Comment by SimonWoolf** [comment]
@paddybyers removed the regen commit and squashed the remaining ones together, pushed to [until-attach-2](https://github.com/ably/ably-js/compare/until-attach-2?expand=1). I'll stop adding regenerate in PRs.

**Comment by paddybyers** [comment]
Thanks

**Pull request closed without merging**